src/sentry/spans/grouping/strategy/base.py

```diff
@@ -119,6 +119,22 @@ def normalized_db_span_in_condition_strategy(span: Span) -> Optional[Sequence[st
     return [cleaned]
 
 
+# Catches sequences like (?, ?, ?), ($1, $2, $3), and (%s, %s, %s)
+LOOSE_IN_CONDITION_PATTERN = re.compile(r" IN \(((%s|\$?\d+|\?)(\s*,\s*(%s|\$?\d+|\?))*)\)", re.I)
+
+
+@span_op(["db", "db.query", "db.sql.query", "db.sql.active_record"])
+def loose_normalized_db_span_in_condition_strategy(span: Span) -> Optional[Sequence[str]]:
+    """This is identical to the above
+    `normalized_db_span_in_condition_strategy` but it uses a looser regular
```

Comment:
I would probably explain more broadly in this comment and not include the `above` portion so if (when) we delete the old one we don't also have to rewrite this comment.